templates/footer-block.html

```diff
@@ -9,13 +9,19 @@
                     © 2018 Copyright
                 </p>
             </div>
-            <div class="col l4 offset-l2 s12">
+            <div class="col l2 offset-l2 s12">
                 <h5 class="white-text">Quick Links</h5>
                 <ul>
-                    <li><a class="grey-text text-lighten-3" href="#!">We are Open Source</a></li>
-                    <li><a class="grey-text text-lighten-3" href="#!">Your history</a></li>
-                    <li><a class="grey-text text-lighten-3" href="#!">Partners</a></li>
-                    <li><a class="grey-text text-lighten-3" href="https://github.ibm.com/blue-xolo/blue-xolo-framework/wiki">Bluexolo Wiki</a></li>
+                    <li><a class="grey-text text-lighten-3" target="blank" href="https://www.ibm.com/opensource/">We are Open Source</a></li>
+                    <li><a class="grey-text text-lighten-3" target="blank" href="https://github.ibm.com/blue-xolo/blue-xolo-framework/wiki">Bluexolo Wiki</a></li>
```

Comment:
This link to the old wiki is no longer active and I don't think we'll have another soon, please comment this out

Reply:
I talked about this with @Grixtli and the proposed solution is to add the wiki to the current repo and update the anchor reference.